src/openrct2/actions/IGameAction.h

```diff
@@ -68,12 +69,14 @@ struct GameActionResult
     rct_xyz32       Position = { 0 };
     money32         Cost = 0;
     uint16          ExpenditureType = 0;
-    void *          Tag = nullptr;
+    uint64          Results[4];
```

Comment:
This must have been before I realised we were going to create sub classes of `GameActionResult`. You can just store the data in the derived classes. `GameActionResult`s are not serialised so it doesn't need anything complicated.